src/ship/activeSensors.js

```diff
@@ -1,5 +1,5 @@
-import Vector2 from "../helpers/Vector2.js";
 import EMSReading from "./EMSReading.js";
+import response from "../helpers/response.js";
 
 export default class ActiveSensors{
     constructor(parentShip){
```

Comment:
(This comment applies to line 10 -- first line in the `performScan` function) Return a `Response` object if the solar system does not exists